### ISSUE TITLE
[Feature request] Add support for whole album FLAC files.

### ISSUE THREAD

**Issue by MegaArthur** [AdrienPoupa/VinylMusicPlayer#64]
When you buy albums, they are sometimes distributed as one big (>200MB) FLAC file, which contains each song and a .cue file with info where each song starts and ends.

Some players have no problem playing it, eg. Poweramp, but Vinyl doesn't treat the files separetely, instead as one big file.

**Comment by pdasen** [comment]
Totally agree, for me this feature is essential, as I don't want to have to maintain a separate media library for my phone. There are tons of open source music players on android, sadly none seem to support this feature (only closed source apps do), except one that has been abandoned by the developer. I'm not an android developer but afaik the reason that it's missing in almost every app is that one would have to roll ones own media scanning implementation (as seen implemented for example in the GoneMad Music player app), because the official API (Android.Provider.MediaStore I believe) does not support .cue sheets.

**Comment by Atrate** [comment]
Bump, useless bot.

**Comment by AdrienPoupa** [comment]
I agree it's not the best way to approach this... but with 75 issues open I just need to declutter things a bit :)

**Comment by ChrisMrX** [comment]
Support for built-in CUE files is essential for me too. Currently i have to use another audio player for my FLAC library.

**Comment by smichel17** [comment]
@AdrienPoupa I would suggest grouping issues into related feature sets that you plan to work on together, using either labels, milestones, or cards on a "project" (aka kanban). Then chose priorities at this higher level, where there is less clutter. (Not my original idea -- taken from <https://apenwarr.ca/log/20171213>.)

**Comment by 0x6273** [comment]
Bump. I need this because many of my albums are flac+cue.

**Comment by daemonspudguy** [comment]
Bump so the bot doesn't decide this doesn't matter.

**Comment by bernharl** [comment]
I second this!

**Comment by MintMain21** [comment]
Don't mind me, just commenting to keep the issue from going.....stale.

**Comment by Sneethe** [comment]
I was excited from the 'Vinyl' name that .cue would most likely be supported. Disappointed to find out that was not the case.

**Comment by krllschmdt** [comment]
Why does this bot even exist?

**Comment by MintMain21** [comment]
At this point, I'm convinced this project is abandoned. I've moved on to Retro Music Player and I reccomend everyone do the same.

**Comment by MintMain21** [comment]
Adrian, if you can read this, your Music Player is still the best on Android and it deserves some love.

**Comment by xeYe-org** [comment]
I'd also like to see cue sheet support arrive (for different file types though).
